R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2252,6 +2252,31 @@ test_that("Method str()", {
   expect_equal(capture.output(utils:::str(iris)), capture.output(str(iris)))
 })
 
+test_that("collect/show/head on Columns", {
```

Comment:
I think if we remove `collect()` from this PR the test needs to be updated as well.